src/super_gradients/recipes/dataset_params/imagenet_dataset_params.yaml

```diff
@@ -12,3 +12,5 @@ imagenet_pca_aug: 0.0 # imagenet pca augmentation (applied only to trainset)
 train_interpolation: default # interpolation mode
 rand_augment_config_string: # randaugment config string (see super_gradients/training/datasets/auto_augment.py)
 random_erase_prob: 0.0 # random erase probability (applied only to trainset)
+use_dali: False # whether to use dali pipeline
```

Comment:
@shaydeci  - Let's be a little more explicit for this param...
 Consider - "nvidia_dali_data_loading" or something else, so it will be clear to our users...